src/components/Combobox/Combobox.tsx

```diff
@@ -222,6 +228,9 @@ export const Combobox: React.FC<ComboboxProps> = ({
           })}
         </div>
       </div>
+      <div role="region" id={`${id}-listbox`} aria-live="polite" className="visually-hidden">
+        {useAriaLive ? ariaLive : ''}
+      </div>
```

Comment:
Could use a timer but it can be difficult to choose a specific time that will be sufficient enough for the text to be read and at the same time not last too long. That's why I made it disappear onBlur for now.